<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ejemplo</class>
 <widget class="QDialog" name="Ejemplo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ejemplo</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>211</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="nombreL">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nombreLE"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="correoL">
      <property name="text">
       <string>Correo:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="correoLE"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="contraL">
      <property name="text">
       <string>Contase√±a:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="contraLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="guardarB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
